legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -619,6 +620,26 @@ def alter_outputs_special_resolution(filing, outputs):
                 outputs.remove('certifiedRules')
         return outputs
 
+    @staticmethod
+    def get_static_documents(filing, url_prefix):
+        """Get static documents."""
+        outputs = []
+        if filing.filing_type == 'continuationIn':
+            continuation_in = filing.meta_data.get('continuationIn', {})
+            if file_key := continuation_in.get('affidavitFileKey'):
+                outputs.append({
+                    'name': 'Director Affidavit',
```

Comment:
"Director" or "Director's" ? I've seen both. Please check with Andy.

Reply:
This shows "Director Affidavit":

![image](https://github.com/bcgov/lear/assets/10002889/370574ed-8c9d-4311-94ea-0090fba19d57)
https://www.figma.com/design/wVF8EcUCUR20YyqA0wHVJp/Continuation-In-V5?node-id=56-5091&t=9KjFzGFK37HPQIKH-0

Reply:
This shows a few options:

![image](https://github.com/bcgov/lear/assets/10002889/d48fba5c-8f23-42b0-9388-03160f8305c3)
https://www.figma.com/design/wVF8EcUCUR20YyqA0wHVJp/Continuation-In-V5?node-id=56-2768&t=9KjFzGFK37HPQIKH-0

Reply:
"Director Affidavit". Confirmed with Andy